index.js

```diff
@@ -40,7 +43,35 @@ function toggleColor(element) {
   }
 }
 
+document.addEventListener('click', event => {
+  if (event.target === header) {
+    toggleColor(header)
+  }
+  else if (event.target.className === 'like-button') {
+    addLikes(event)
+
+  }
+})
```

Comment:
A couple notes on this one:

- Be careful of using event delegation on the entire document to handle all your events; this will work, but it can get tricky to manage this code as your application grows. It's preferable to add event listeners to the closest parent element if you're using delegation (for the `.like-button` elements, the closest parent is the `..player-container`)
- Small note on code style: `else` statements are usually written on the same line as the end of the `if` block, like this:

```js
  if (event.target === header) {
    toggleColor(header)
  } else if (event.target.className === 'like-button') {
    addLikes(event)
  }
```